requirements.txt

```diff
@@ -7,3 +7,4 @@ scipy>=1.4.0
 seaborn>=0.9.0
 joblib>=0.11
 python-picard >= 0.4
+multiviewica >= 0.0.1
```

Comment:
remove

Reply:
I would like to keep multiviewica as a dependency for now. At some point, I'll make another pull request to move multiviewICA here but for the time being I think it is simpler if we do not duplicate the implementations.

Reply:
a dependency in a software is not cheap. If we use multiviewica I think we should not have an implementation in mvlearn. It should be a soft dependency with a nested import. Let's avoid code duplication